### DISCUSSION TITLE
BrainVision .ahdr files

### DISCUSSION THREAD

**Discussion by spressi** [bnicenboim/eeguana#162] [General]
Hello everybody,
First of all: thank you for this great R package! I used it to process electrodermal data measured with a [BrainAmp](https://brainvision.com/products/brainamp-standard/) system (stored as .vhdr) and it worked great! In another project, I am now trying to open physiological data (EDA + ECG) measured with a [V-Amp](https://brainvision.com/products/v-amp/) (stored as .ahdr). Using `read_vhdr`, this seems to work, the markers are fine but the signal is gibberish. 
According to [this post](https://sccn.ucsd.edu/pipermail/eeglablist/2013/007301.html), the .eeg data referred to in .ahdr files are encrypted and need to be exported to .txt with the BrainVision Analyzer. If you use a BrainAmp, you should be able to choose the file format, so make sure to select .vhdr!
Best regards,
Mario

**Comment by bnicenboim** [comment]
Here the developer, I'm glad it worked out!
